tfrog-motordriver/main.c

```diff
@@ -282,10 +287,37 @@ void timer1_tic()
   }
 #endif
 }
+
+void timer2_cb()
+{
+  volatile unsigned int dummy2;
+  dummy2 = AT91C_BASE_TC2->TC_SR;
+  dummy2 = dummy2;
+
+#if defined(tfrog_rev5)
+  if (heartbeat_timeout < 0)
+    return;
+
+  if (heartbeat_timeout < 5)
+  {
+    heartbeat_timeout++;
+  }
+  else
+  {
+    PIO_Clear(&pinMotor);
+    printf("heartbeat timeout\n\r");
```

Comment:
please add error ID and indicator LED blink pattern
https://github.com/T-frog/tf-2md3-firmware/blob/f9666a8655c0627d5fa409f4530eaea4484ada10/tfrog-motordriver/controlVelocity.h#L29-L37
https://github.com/T-frog/tf-2md3-firmware/blob/f9666a8655c0627d5fa409f4530eaea4484ada10/tfrog-motordriver/errors.h#L21-L37

, and set error_state 
https://github.com/T-frog/tf-2md3-firmware/blob/f9666a8655c0627d5fa409f4530eaea4484ada10/tfrog-motordriver/main.c#L1068-L1069